pkg/sql/executor_ir.go

```diff
@@ -163,6 +164,16 @@ func writeArgoFile(coulerFileName string) (string, error) {
 	return argoYaml.Name(), nil
 }
 
+func getWorkflowID(output string) (string, error) {
```

Comment:
Duplicated with `workflow.go`#getWorkflowID ?

Reply:
Yes, I will move workflow related code to `package workflow` in the next PR.